cmd/skopeo/proxy.go

```diff
@@ -600,6 +600,7 @@ func (h *proxyHandler) GetBlob(args []any) (replyBuf, error) {
 		tr := io.TeeReader(blobr, verifier)
 		n, err := io.Copy(f.w, tr)
 		if err != nil {
+			logrus.Debugf("")
```

Comment:
Do you want to add some distinct text here?

Reply:
Forgot to amend the commit after vacillating on whether a debug log is needed there; I don't think it is, so deleted.